index.html

```diff
@@ -189,9 +186,34 @@ <h2 class=" connect">LET'S CONNECT</h2>
                 <a href="#" class="dropdown-btn"><div>Skills</div><img src="images/arrow.png"></a>
             </li>
           </ul>
-       
+        </div>
     </section>
-    
+    <section class="contact-me">
+    
+        <div class="form-header">
+        <h2 class="form-heading"> Contact me </h2>
+ 
+        <p class="pg-contact">If you have an application you are interested in developing, a feature that you
+          need built or a project that needs coding. 
+         I’d love to help with it!
+         </p>
+         </div>
+         <form action="https://formspree.io/f/mqkvkgdo" method="POST" class="contact-me-form">
+         <ul class="form-info">
+         <li>     
+             <input type="text" id="name" name="name" required placeholder="enter your name" maxlength="30"/></li>
+         <li>
+                 <input type="email" id="e-mail" name="email" required placeholder="enter your email"/> </li>
```

Comment:
- [ ] As mentioned by the previous reviewer, please make sure the form is not submitted if one of the input fields is empty. I remarked that this issue is still not addressed yet